models/Role.js

```diff
@@ -0,0 +1,41 @@
+/**
+ * A model class representing roles
+ *
+ * @param {Sequelize} sequelize - Sequelize object
+ * @param {Sequelize.DataTypes} DataTypes - A convinient object holding data types
+ * @return {Sequelize.Model} - Role model
+ */
+export default (sequelize, DataTypes) => {
+  const Role = sequelize.define('Role', {
+    title: {
+      allowNull: false,
+      type: DataTypes.STRING,
+    },
+    create: {
+      type: DataTypes.BOOLEAN,
+      defaultValue: false
+    },
+    read: {
```

Comment:
#### Critical 
Use this [stackoverflow thread](https://stackoverflow.com/questions/46016139/best-user-role-permissions-database-design-practice) to understand how to implement permissions properly.

Reply:
Thanks for the resource, I'll make improvements on the role.

Reply:
Hi @aanchirinah, I have extended the roles with support for multiple permissions.